test/shared/src/main/scala/zio/test/Spec.scala

```diff
@@ -312,6 +312,17 @@ final case class Spec[-R, +E, +L, +T](caseValue: SpecCase[R, E, L, T, Spec[R, E,
         val (z, caseValue) = f(z0, t)
         ZIO.succeed(z -> Spec(caseValue))
     }
+
+  /**
+   * Runs only tests whose labels (which must be strings) contain the given substring
+   */
+  final def only[S, E1](
+    s: String
+  )(implicit ev1: L <:< String, ev2: T <:< TestSuccess[S], ev3: E <:< TestFailure[E1]): ZSpec[R, E1, String, S] =
```

Comment:
Sorry, to be consistent can we flip the order here so the `TestFailure` evidence comes before the `TestSuccess` evidence?